pinot-broker/src/main/java/org/apache/pinot/broker/queryquota/HelixExternalViewBasedQueryQuotaManager.java

```diff
@@ -353,6 +461,11 @@ public int getRateLimiterMapSize() {
     return _rateLimiterMap.size();
   }
 
+  @VisibleForTesting
```

Comment:
nit: for the two methods marked VisibleForTesting, perhaps just add a method (VisibleForTesting) to return _databaseRateLimiterMap

Reply:
Will do